README.md

```diff
@@ -279,13 +279,15 @@ function someFunction()
 const count = 2;
 (function doSomething() {
   // do something amazing
-}());
+})();
 
 // Avoid — throws exception
-const count = 2 // it tries to run 2(), but 2 is not a function
-(function doSomething() {
-  // do something amazing
-}())
+const count = 2(
```

Comment:
Something broke here, guess it's due to the missing semicolon which shifted it up

Reply:
Good catch, @BrunoScheufler! 👍 

I think prettier auto-formatted the code blocks following my local config.

Let me comb through the entire file again to see if there are more such instances.